.prettierignore

```diff
@@ -4,3 +4,4 @@ tests/internal/testopenai/cassettes
 tests/internal/testopeninference/spans
 cmd/aigw/testdata
 internal/autoconfig/testdata
+examples/aigw/ollama.yaml
```

Comment:
This file is used in the autoconfig test, and we want it to match the template output. I'm ignoring the formatting for it, as I don't wanna couple the autoconfig generation template to the yaml pretty formatting.